fili-core/src/main/java/com/yahoo/bard/webservice/config/BardFeatureFlag.java

```diff
@@ -7,8 +7,9 @@
  */
 public enum BardFeatureFlag implements FeatureFlag {
     PARTIAL_DATA("partial_data_enabled"),
-    DRUID_CACHE("druid_cache_enabled"),
+    @Deprecated DRUID_CACHE("druid_cache_enabled"),
     DRUID_CACHE_V2("druid_cache_v2_enabled"),
```

Comment:
I think Druid cache v2 will still be supported for now because we cannot count on fili users being on a particular version of druid and we haven't yet got a back-end-version strategy for config other than continuing to support necessary features.

In the case of V2, all the interesting druid versions support it.  In the case of eTag support, I'm not sure which versions support it, but not all of the ones since our launch.

We can deprecate it, but not until we have a clear policy around which versions of druid are supported.